command-launcher-plugin/pom.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.jenkins-ci.plugins</groupId>
+        <artifactId>plugin</artifactId>
+        <version>2.34</version>
+        <relativePath />
+    </parent>
+    <artifactId>command-launcher</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <packaging>hpi</packaging>
+    <properties>
+        <jenkins.version>2.85-SNAPSHOT</jenkins.version> <!-- TODO pending split -->
+        <java.level>8</java.level>
+    </properties>
+    <name>Command Launcher Plugin</name>
```

Comment:
Command Computer Launcher Plugin? Just to avoid confusion with other launcher types